frontend/package.json

```diff
@@ -23,13 +23,14 @@
   },
   "dependencies": {
     "@next/third-parties": "^14.0.4",
-    "@trussworks/react-uswds": "^5.0.0",
+    "@trussworks/react-uswds": "^7.0.0",
     "@uswds/uswds": "^3.6.0",
     "i18next": "^23.0.0",
     "js-cookie": "^3.0.5",
     "lodash": "^4.17.21",
-    "next": "^13.5.2",
+    "next": "^14.0.3",
```

Comment:
@acouch was this required for anything? Or we just trying to get to the latest before implementing search

Reply:
There were errors that this fixed. Next jumped from [13.5](https://github.com/vercel/next.js/releases/tag/v13.5.6) to [14.0](https://github.com/vercel/next.js/releases/tag/v14.0.0) so the major jump was inevitable. This also keeps us up-to-date with the template.